### PR TITLE
Add execution error and latency metrics

### PR DESCRIPTION
**Pull request by mpetrun5**
Implement deposit error rate and deposit execution latency metrics.

## Related Issue Or Context

Closes: #<issue>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in chainbridge-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

**Review by MakMuftic — APPROVED** [review]

**Review by mpetrun5 — COMMENTED** [review]
Will add some more labels for domains  (so we can possibly even filter by source and destination) and setup in docker-compose.yml to erase ConsoleTelemetry.

File: relayer/relayer.go [hunk 1]
```diff
@@ -75,7 +77,19 @@ func (r *Relayer) route(msgs []*message.Message) {
 	}
 
 	log.Debug().Msgf("Sending messages %+v to destination %v", msgs, destChain.DomainID())
-	destChain.Write(msgs)
+
+	err := destChain.Write(msgs)
```
Comment (MakMuftic) [inline relayer/relayer.go:81]: Isn't `destChain.Write(msgs)` always returning `nill`?

Comment (MakMuftic) [inline relayer/relayer.go:81]: Isn't `destChain.Write(msgs)` always returning `nill`?

Comment (mpetrun5) [reply relayer/relayer.go:81]: In this implementation it is.

**Review by nmlinaric — APPROVED** [review]

**Pull request merged**